src/SecurityManager.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kevin;
+
+class SecurityManager
+{
+    /**
+     * Endpoint secret.
+     *
+     * @var string
+     */
+    private $endpointSecret;
+
+    /**
+     * SecurityManager constructor.
+     *
+     * @param string $endpointSecret
+     */
+    public function __construct($endpointSecret = '')
+    {
+        $this->endpointSecret = $endpointSecret;
+    }
+
+    /**
+     * An implementation of kevin.'s signature.
+     * @see https://docs.kevin.eu/public/platform/v0.3#tag/Signature
+     *
+     * @param string $requestBody
+     * @param array $headers
+     * @param string $webhookUrl
+     * @param float $timestampTimeout in milliseconds
+     * @return bool
+     */
+    public function verifySignature($requestBody, $headers, $webhookUrl, $timestampTimeout = null)
+    {
+        if (!$this->verifyTimeout($timestampTimeout, $headers)) {
+            return false;
+        }
+
+        $data = 'POST' . $webhookUrl . $headers['X-Kevin-Timestamp'] . $requestBody;
+        $signature = $this->generateHasWithHmac256($data, $this->endpointSecret);
+
+        return $signature == $headers['X-Kevin-Signature'];
+    }
+
+    /**
+     * Generate HMAC SHA256 hash.
+     *
+     * @param string $data
+     * @param string $key
+     * @return string
+     */
+    private function generateHasWithHmac256($data, $key)
```

Comment:
Not sure if it's worth to create a separate method if it's only used in one place.